lib/cli/scripts/changelog.ts

```diff
@@ -113,18 +111,21 @@ function getCommits(options: DiffOptions): Array<Commit> {
     // https://stackoverflow.com/a/13928240/14644447
     log = log.trim().replace(/"/gm, '\\"').replace(/\^@\^/gm, '"');
     if (log.endsWith(',')) {
-        log = log.substring (0, log.length - 1);
+        log = log.substring(0, log.length - 1);
     }
 
-    return log.split('\n').map(str => JSON.parse(str) as Commit);
+    return log.split('\n').map(str => JSON.parse(str) as Commit).filter(commit => {
+        const filterRegex = RegExp(authorFilter);
+        return !(filterRegex.test(commit.author) || filterRegex.test(commit.author_email));
+    });
 }
 
 export default function main(_args: string[], workingDir: string) {
     program
         .description('Generate changelog report for two branches of git repository')
         .version('0.0.1', '-v, --version')
         .usage('changelog [options]')
-        .option('-r, --range <range>', 'Commit range, e.g. master..develop', 'master..develop')
+        .option('-r, --range <range>', 'Commit range, e.g. origin/master..develop', 'origin/qmaster..develop')
```

Comment:
is origin needed here? was working without that too

Reply:
if you use a perl version of git it won't work without `origin/` so I thought it would be better to add it as it works with both compilations of git. I can remove it if you want, no problem from my side.